### ISSUE TITLE
Add GitHub topics to repository for easier discovery

### ISSUE THREAD

**Issue by fjeremic** [eclipse/openj9#5034]
We can take advantage of GitHub's topics feature [1] to add [2] the Eclipse OpenJ9 to a bunch of topics which will effectively increase awareness of our project for people using the topics feature to search repositories relating to their search term.

[1] https://github.com/topics/
[2] https://help.github.com/en/articles/classifying-your-repository-with-topics

**Comment by dsouzai** [comment]
+1

**Comment by pshipton** [comment]
I suggest using words instead of acronyms, unless the acronym already exists as a topic.
i.e.
Garbage Collector
Virtual Machine

**Comment by fjeremic** [comment]
Taking a look at some similar projects I'm seeing both "vm", "virtual-machine". There is not much for "gc" but some hits for "garbage-collector", so we should likely go with those. I'm not seeing almost any for "just-in-time", however "jit" seems to be common.

Projects I've looked at:
https://github.com/mono/mono
https://github.com/oracle/graal
https://github.com/Microsoft/ChakraCore
https://github.com/v8/v8
https://github.com/llvm-mirror/llvm

I've updated the list in the description accordingly.

**Comment by DanHeidinga** [comment]
Thanks @fjeremic - This gets a big +1 from me.  As this will require help from the Eclipse Webmaster, we should think about whether the text at the top of the project page needs updating as well so we can batch the requests

**Comment by ashu-mehra** [comment]
I think it should be ok to have both 'gc' and 'garbage-collector'.

**Comment by fjeremic** [comment]
> As this will require help from the Eclipse Webmaster, we should think about whether the text at the top of the project page needs updating as well so we can batch the requests

I agree. Perhaps we ought to include a link to the website. Giving it some more thought and let's see what we can come up with.
